### PR TITLE
Fix button visibility [SCI-4663]

### PR DESCRIPTION
**Pull request by urbanrotnik**
Jira ticket: [SCI-4663](https://biosistemika.atlassian.net/browse/SCI-4663)

### CODE REVIEW

File: app/views/my_modules/repositories/_full_view_version.html.erb [hunk 1]
```diff
@@ -28,9 +28,11 @@
       <% end %>
     </div>
     <div class="col-xs-4">
-      <a class="pull-right btn version-button delete-snapshot-button" href="#" data-action-path="<%= my_module_repository_snapshot_path(@my_module, repository_snapshot) %>">
-        <i class="fas fa-trash"></i>
-      </a>
+      <% if can_manage_module?(@my_module) %>
```
Comment (okriuchykhin) [inline app/views/my_modules/repositories/_full_view_version.html.erb:31]: This check will be executed separately for each snapshot? Can you move it out from the loop and use local variable?

Comment (urbanrotnik) [reply app/views/my_modules/repositories/_full_view_version.html.erb]: Updated. Anyway, the value of this check is loaded in the cache and it's not querying each iteration

File: app/views/my_modules/repositories/_full_view_sidebar.html.erb [hunk 2]
```diff
@@ -40,13 +40,18 @@
           </button>
         </div>
       <% end %>
-      <%= render partial: 'my_modules/repositories/full_view_version', collection: @repository_snapshots, as: :repository_snapshot %>
+      <%= render partial: 'my_modules/repositories/full_view_version',
+                 collection: @repository_snapshots,
+                 as: :repository_snapshot,
+                 locals: {can_delete_snapshot: can_manage_module?(@my_module)}%>
```
Comment (okriuchykhin) [inline app/views/my_modules/repositories/_full_view_sidebar.html.erb:46]: spaces are missing near { and }

File: app/views/my_modules/repositories/_full_view_sidebar.html.erb [hunk 3]
```diff
@@ -40,13 +40,18 @@
           </button>
         </div>
       <% end %>
-      <%= render partial: 'my_modules/repositories/full_view_version', collection: @repository_snapshots, as: :repository_snapshot %>
+      <%= render partial: 'my_modules/repositories/full_view_version',
+                 collection: @repository_snapshots,
+                 as: :repository_snapshot,
+                 locals: { can_delete_snapshot: can_manage_module?(@my_module) }%>
```
Comment (okriuchykhin) [inline app/views/my_modules/repositories/_full_view_sidebar.html.erb:46]: Space is missing after }

**Pull request merged**